pkgs/development/python-modules/tidalapi/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, stdenv
+, buildPythonPackage
+, fetchPypi
+, python-dateutil
+, requests
+}:
+
+buildPythonPackage rec {
+  pname = "tidalapi";
+  version = "0.7.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-LdlTBkCOb7tXiupsNJ5lbk38syKXeADvi2IdGpW/dk8=";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    python-dateutil
+  ];
+
+  doCheck = false;
```

Comment:
Tidalapi has tests on the Github repository. So, you need to pull it from GitHub. And in case the tests cannot be applied because of the sandboxing, you need to specify why, in a comment, the tests are disabled.